liquibase-core/src/main/resources/liquibase/examples/start-h2

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+if [ -z "${LIQUIBASE_HOME}" ]; then
```

Comment:
Is this file supposed to be here @wwillard7800 ?

Reply:
@StevenMassaro If this file belongs here then should be updated with latest h2 jar file which is h2-2.0.202.jar

Reply:
I'll update the script